file_paths.md

```diff
@@ -1,24 +1,341 @@
+                    LC_MESSAGES *
+                        <project>.mo *    # One directory for each supported locale,
+                                          # containing the MO files for each project.
+                                          # Example for English:
+                                          # /opt/puppetlabs/puppet/share/locale/en_US/LC_MESSAGES/puppet.mo
+        ssl *
+        vendor_modules *                  # :vendormoduledir
+        VERSION                           # puppet-agent package version
+
+    /opt/puppetlabs/pxp-agent *
+        modules *
+            pxp-module-puppet *
+        spool *                            # directory containing results of pxp-agent modules
+        tasks-cache *                      # directory containing a cache of files for all downloaded tasks
+
+    /var/log/puppetlabs *
+        puppet *                          # :logdir                      /var/lib/puppet/log
+            puppet.log                    # not enabled by default
+        pxp-agent *
+            pxp-agent.log                 # enabled by default
+            pcp-access.log                # not enabled by default
+
+    /var/run/puppetlabs *                 # :rundir                      /var/lib/puppet/run
+        agent.pid                         # :pidfile
+        pxp-agent.pid
+
+# puppet-agent 6 (Windows)
+
+On Windows:
+
+- The installation path defaults to `C:\Program Files\Puppet Labs\Puppet`
+    - When installing puppet-agent 32-bit on 64-bit Windows, the default
+      installation path is `C:\Program Files (x86)\Puppet Labs\Puppet`
+- The common app data directory defaults to `C:\ProgramData\PuppetLabs`
+
+The package will create the following services, all running as
+`LocalSystem`, by default:
+
+- `puppet`,
+- `pxp-agent`
+
+It will not create a `puppet` user or group.
+
+The structure of the Puppet 6 agent on Windows differs from Puppet 5 in that:
+
+- `C:\Program Files\Puppet Labs\Puppet\puppet\vendor_modules` has been added to
+  hold core puppet modules
+- mcollective and all of its associated files have been removed
+- To make the paths in the Windows agent more consistent with other platforms,
+  install paths have changed as follows:
+    - The base of the ruby installation has moved from
+      `C:\Program Files\Puppet Labs\Puppet\sys\ruby` to
+      `C:\Program Files\Puppet Labs\Puppet\puppet`
+    - Tools (like `elevate.exe`) have similarly moved from
+      `C:\Program Files\Puppet Labs\Puppet\sys\tools` to
+      `C:\Program Files\Puppet Labs\Puppet\puppet\bin`
+    - The agent components facter, hiera, and pxp-agent no longer have
+      their own dedicated install locations at
+      `C:\Program Files\Puppet Labs\Puppet\<component-name>`; they are installed
+      to the same prefix as other projects (`C:\Program Files\Puppet
+      Labs\Puppet\puppet`)
+
+The files and directories annotated by an '*' below indicate that they
+are created by package installation.
+
```

Comment:
Perhaps add a note that the paths are the Windows default installation paths and may very if Windows has been customised (or words to that affect)